app/javascript/components/Product/Card.tsx

```diff
@@ -37,61 +41,81 @@ export const Card = ({
         />
       ) : null}
       {product.ratings?.count ? <Rating ratings={product.ratings} /> : null}
-    </header>
-    <footer>
-      <PriceTag
-        url={product.url}
-        currencyCode={product.currency_code}
-        price={product.price_cents}
-        isPayWhatYouWant={product.is_pay_what_you_want}
-        isSalesLimited={product.is_sales_limited}
-        recurrence={
-          product.recurrence ? { id: product.recurrence, duration_in_months: product.duration_in_months } : undefined
-        }
-        creatorName={product.seller?.name}
-      />
+    </ProductCardHeader>
+    <ProductCardFooter>
+      <div className="flex-1 p-4">
+        <PriceTag
+          url={product.url}
+          currencyCode={product.currency_code}
+          price={product.price_cents}
+          isPayWhatYouWant={product.is_pay_what_you_want}
+          isSalesLimited={product.is_sales_limited}
+          recurrence={
+            product.recurrence ? { id: product.recurrence, duration_in_months: product.duration_in_months } : undefined
+          }
+          creatorName={product.seller?.name}
+        />
+      </div>
       {footerAction}
-    </footer>
+    </ProductCardFooter>
     {badge}
-  </article>
+  </ProductCard>
 );
 
 export const HorizontalCard = ({ product, big, eager }: { product: CardProduct; big?: boolean; eager?: boolean }) => (
-  <article className={cx("product-card horizontal relative", { big })}>
-    <figure>
+  <ProductCard className="lg:flex-row">
+    <ProductCardFigure className="lg:h-full lg:rounded-l lg:rounded-tr-none lg:border-r lg:border-b-0 [&_img]:lg:h-0 [&_img]:lg:min-h-full lg:[&_img]:w-auto">
```

Comment:
I don't like how much we're using descendant selectors here, but they're necessary to override the ones in ProductCardFigure. I won't hold this up longer but would be nice to find a better solution at some point (maybe when we're fully migrated and have more examples/experience)